From 5dfdaf9a18971bf5504a0d3fe00f747a853ee800 Mon Sep 17 00:00:00 2001
From: Tomas Bzatek <tbzatek@redhat.com>
Date: Thu, 01 Nov 2012 13:46:10 +0000
Subject: proxy volume monitor: Don't force-close the session bus

Simple fix to prevent bus closing on GIO module unload.

https://bugzilla.gnome.org/show_bug.cgi?id=687074
---
diff --git a/monitor/proxy/gproxyvolumemonitor.c b/monitor/proxy/gproxyvolumemonitor.c
index e0d17b7..df99935 100644
--- a/monitor/proxy/gproxyvolumemonitor.c
+++ b/monitor/proxy/gproxyvolumemonitor.c
@@ -1435,7 +1435,6 @@ g_proxy_volume_monitor_teardown_session_bus_connection (void)
   G_LOCK (proxy_vm);
   if (the_session_bus != NULL)
     {
-      g_dbus_connection_close_sync (the_session_bus, NULL, NULL);
       g_object_unref (the_session_bus);
       the_session_bus = NULL;
 
--
cgit v0.9.0.2
